go.mod

```diff
@@ -7,6 +7,8 @@ require (
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/julienschmidt/httprouter v1.3.0
+	github.com/libsv/bitcoin-hc v0.3.0
```

Comment:
We should rather use our structures instead use whole Pulse as a dependency

Reply:
Done